src/libraries/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.Create.cs

```diff
@@ -459,7 +459,7 @@ public static ZipArchive Open(string archiveFileName, ZipArchiveMode mode, Encod
                 throw new ArgumentOutOfRangeException(nameof(compressionLevel));
             }
 
-            // Rely on Path.GetFullPath for validation of sourceDirectoryName and destinationArchive
+            // Rely on Path.GetFullPath for validation of sourceDirectoryName
 
             // Checking of compressionLevel is passed down to DeflateStream and the IDeflater implementation
             // as it is a pluggable component that completely encapsulates the meaning of compressionLevel.
```

Comment:
Thanks for noticing. Yes, I think we can also remove the compressionLevel comment.

```suggestion
```